README.md

```diff
@@ -1 +1,3 @@
-# ProjectA
\ No newline at end of file
+# ProjectA
+
+feature1 - origin selection
```

Comment:
Line No 3
Can be optimised

Reply:
addressed in 1d9d919473ed7993534f63fe9fda0d3202eaad06